code/modules/mob/living/carbon/life.dm

```diff
@@ -665,6 +665,9 @@
  * * capped (optional) default True used to cap step mode
  */
 /mob/living/carbon/adjust_bodytemperature(amount, min_temp=0, max_temp=INFINITY, use_insulation=FALSE, use_steps=FALSE, capped=TRUE)
+	var/datum/status_effect/void_chill/freeze_effect = has_status_effect(/datum/status_effect/void_chill)
+	if(freeze_effect?.stacks >= 5 && amount > 0)
+		return
```

Comment:
This hard coded check in the middle of body temp code is icky, can we make it a signal or trait instead